Formula/jbang.rb

```diff
@@ -1,25 +1,24 @@
 class Jbang < Formula
-  desc "Tool to create, edit and run self-contained source-only Java programs"
-  homepage "https://jbang.dev/"
+  desc "Unleash the power of Java"
+  homepage "https://jbang.dev"
   url "https://github.com/jbangdev/jbang/releases/download/v0.102.0/jbang-0.102.0.zip"
+  version "0.102.0"
   sha256 "74f9ec600ac193e158221b3f5b95e156349ace730ed841eab39e2b1205b48622"
   license "MIT"
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "7055856af09e18b5108eb8d2d1846e9d2b1874bde9c984deb4fc466a6a523a04"
   end
 
-  depends_on "openjdk"
-
   def install
-    libexec.install "bin/jbang.jar"
-    bin.write_jar_script libexec/"jbang.jar", "jbang"
+    libexec.install Dir["*"]
+    inreplace "#{libexec}/bin/jbang", /^abs_jbang_dir=.*/, "abs_jbang_dir=#{libexec}/bin"
```

Comment:
This script will download its own Java runtime if it can't find one, which we want to avoid.

What is missing from the existing jar script?

Reply:
Jbang is the actual script + the jar. 

We Got multiple reports by users getting jbang not running and Thats because This brew script completelty slips the actual execution of the command jbang generetes to run. 

Why do You prefer Users having to download a Java they dont need? Jbang Works with Any Java 8 and onward and users can declare to use Any Java version to execute with. 

Thus Installing openjdk 19 is redundant at best. 

You can do it but I find it counter to what makes sense when the feature of jbang is to be able to use what already installed and only get Java if necessary.

Reply:
If There is way for brew to only have it depend on openjdk 19 if No java in path that would be Nice?

Reply:
Brew formula must always depend on brew dependencies. If a tool depends on Java it must therefore add `depends_on "openjdk"` or optionally `@8,11 or 17`.

Reply:
That's a shame. 

Can one express it can either be 8, 11 or 17 so users aren't forced to install something they already have?

Reply:
No, homebrew software should always use the latest version of dependencies so it should just be `depends_on "openjdk"` if that version is compatible.

Reply:
So if users already have openjdk 11 and jbang depends on just openjdk will it force install openjdk 19 ?

jbang works with java 8 and upwards and will so in any foreseeable future.

Reply:
added back "openjdk"